resources/migrations/001_update_migrations.yaml

```diff
@@ -5876,6 +5876,619 @@ databaseChangeLog:
         - dbms:
             type: mysql,mariadb
 
+  - changeSet:
+      id: v49.2024-05-29T09:26:20
+      author: SakuragiYoshimasa
```

Comment:
@johnswanson I think it might be more useful to have your name on these migrations as well.

Not to take credit away from @SakuragiYoshimasa (thanks for your contribution!) but the main use of the 'author' field in my experience is to figure out who (or what team) to ping if a migration fails. So it's better to have a Metabase dev on here. (Not that I'd expect these to fail, but ya know, just in case)